<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChapterSplit - Bookshot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="booksLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="refreshBookBtn">
            <property name="text">
             <string>Refresh Book</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="bookListWidget">
          <property name="baseSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="saveBookInfoBtn">
          <property name="text">
           <string>Save Book Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="coverImgLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="upCoverBtn">
          <property name="text">
           <string>Update Cover</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>nid</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nidEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>bookName</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="bookNameEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>author</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="authorEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="typeEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>createTime</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="createTimeEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>desc</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="descriptionEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="saveChapterBtn">
            <property name="text">
             <string>Save Chapter Info</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Page Now/All</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="nowPageLabel">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="allPageLabel">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLineEdit" name="pageEdit">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pageBtn">
            <property name="text">
             <string>Go</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="previousBtn">
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextBtn">
            <property name="text">
             <string> Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QPushButton" name="refreshChapterBtn">
            <property name="text">
             <string>Refresh Page</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearBtn">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLineEdit" name="chapNumEdit">
            <property name="text">
             <string>12</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="autoSplitBtn">
            <property name="text">
             <string>AutoSplit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListWidget" name="completeListWidget"/>
          <widget class="QListWidget" name="chapListWidget">
           <property name="baseSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="imgTableWidget">
       <property name="baseSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
